Remove isdn_cdb.h from TARGETS, so it won't be recompiled again. This is unnecessary
because it is already being made explicitly.
Remove isdn_cdb.h from CLEANFILES. The template uses make clean to clean junk from
isdn_cdb.h, but it needs isdn_cdb.h, so this will not delete it.
--- a/src/isdn/cdb/Makefile	2021-10-01 13:55:16.000000000 +0200
+++ b/src/isdn/cdb/Makefile	2021-10-26 12:59:56.810695582 +0200
@@ -1,6 +1,6 @@
 TOPDIR		= ../../..
-TARGETS		= mk_isdnhwdb isdn_cdb.h ISDN.CDB.hwdb
-CLEANFILES	= isdn_cdb isdn_cdb.h lex.yy.c ISDN.CDB.txt mk_isdnhwdb ISDN.CDB.hwdb
+TARGETS		= mk_isdnhwdb ISDN.CDB.hwdb
+CLEANFILES	= isdn_cdb lex.yy.c ISDN.CDB.txt mk_isdnhwdb ISDN.CDB.hwdb
 
 include $(TOPDIR)/Makefile.common
 
@@ -28,6 +28,6 @@
 
 mk_isdnhwdb.o: mk_isdnhwdb.c cdb_read.h lex.yy.c
 
-ISDN.CDB.hwdb: mk_isdnhwdb ISDN.CDB.txt
+ISDN.CDB.hwdb: ISDN.CDB.txt
 	rm -f  ISDN.CDB.hwdb
-	./mk_isdnhwdb ISDN.CDB.txt ISDN.CDB.hwdb
+	mk_isdnhwdb ISDN.CDB.txt ISDN.CDB.hwdb
